### ISSUE TITLE
Working with side-party projects without eslint

### ISSUE THREAD

**Issue by YuriGor** [SublimeLinter/SublimeLinter-eslint#285]
Hi, I know there are several closed issues related to this one, but nothing helped me.

Given:
- I fork some existing side-party project to make some changes and propose a PR
- Project doesn't use eslint at all (no eslint dev dep, no eslint config)
- I am using ST3 + SublimeLinter-eslint (linux)
- on each save I see error about /usr/bin/eslint not found
- Ok, I don't want to, but I install eslint globally
- same error
- Ok, I use nvm, I don't want to do it, but I create symlink at /usr/bin/eslint and point it to nvm/whatever/eslint
- error about no eslint config
- ok, I don't want to do it, but I create "empty" eslint config in the parent dir
- linter highlights false-positive "errors" in this side-party project (e.g. `import` )
- `eslint --reset` doesn't work
- it's very not ok to add something to side-party project just because of my plugin.

Expected:
- if eslint config doesn't exist in the project - it's fine, please, don't lint it.
- if eslint config does exist in the project - yes, it's correct to complain about missing local/global eslint.

As I got it - currently it's impossible, right? 

Will it be possible in the future?

If yes - maybe we need an open issue for this and and explicit declaration of intentions inside, to let people calm down and wait.

If no - please also explicitly confirm this case, to let people just calm down.

If some working workarounds exist - please also share, as I mentioned - I found nothing.

**Comment by kaste** [comment]
We have the `disable_if_not_dependency` setting for that. Basically, in the global SublimeLinter settings. IIRC, (here in yaml instead of JSON)

```
linters:
    eslint:
        disable_if_not_dependency: true
```

**Comment by kaste** [comment]
> on each save I see error about /usr/bin/eslint not found

😕 What exactly is happening here?  It probably shows `(erred)` in the status bar, but it should not open a panel with the default settings.

**Comment by YuriGor** [comment]
Sounds cool, but doesn't work for me.
My config:
```json
// SublimeLinter Settings - User
{
	"paths": {
        "linux": [
        	"/home/gor/.nvm/versions/node/v11.9.0/bin"
        ],
        "osx": [],
        "windows": []
    },
  "linters": {
    "eslint": {
        "fix_on_save": true,
        "disable_if_not_dependency": true
    }
  }
}
```
ST console
```
Using node.js path on 'linux': /home/gor/.nvm/versions/node/v11.9.0/bin/node
Using eslint path on 'linux': /usr/bin/eslint
Unexpected error(<class 'Exception'>): Error: internal/modules/cjs/loader.js:611
    throw err;
    ^

Error: Cannot find module '/usr/bin/eslint'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:609:15)
    at Function.Module._load (internal/modules/cjs/loader.js:535:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:797:12)
    at executeUserCode (internal/bootstrap/node.js:526:15)
    at startMainThreadExecution (internal/bootstrap/node.js:439:3)

error: Error: internal/modules/cjs/loader.js:611
    throw err;
    ^

Error: Cannot find module '/usr/bin/eslint'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:609:15)
    at Function.Module._load (internal/modules/cjs/loader.js:535:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:797:12)
    at executeUserCode (internal/bootstrap/node.js:526:15)
    at startMainThreadExecution (internal/bootstrap/node.js:439:3)

None
```
And exactly the same error as a last one from console is displaying on popup on each save in js file.

**Comment by YuriGor** [comment]
Currently no global eslint installed and no empty eslint config in parent dir. Do I need some?

**Comment by kaste** [comment]
`fix_on_save` is not a valid setting here. And the console output does not look like SublimeLinter.  🤷‍♂

**Comment by YuriGor** [comment]
Oh, sorry, issue comes from another plugin 'ESLint-formatter''
'fix_on_save' - left when I tried that 'fixables' brunch, it has no effect.
Currently, with ESLint-formatter disabled I see no errors with or without 'fix_on_save' or 'disable_if_not_dependency' settings, so sorry again, I will go complaining to another place then.
